assets/queries/dockerfile/disable_pip_cache/test/positive_expected_result.json

```diff
@@ -0,0 +1,7 @@
+[
+	{
+		"queryName": "Disable Pip Cache",
+		"severity": "MEDIUM",
+		"line": 1
```

Comment:
the bug with the docker parser is fixed should be pointing to line 2, not 1.
Merge with the master branch.

Reply:
Fixed in https://github.com/Checkmarx/kics/pull/1410